app/src/components/Secret/Secret.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
I need some component to mount, then token comes. Yulianna will put her component here later.